minard/db.py

```diff
@@ -6,3 +6,10 @@
                                   app.config['DB_HOST'], app.config['DB_PORT'],
                                   app.config['DB_NAME']),
                                   pool_recycle=3600)
+
+nearline_engine = sqlalchemy.create_engine('postgresql://%s:%s@%s:%i/%s' %
```

Comment:
The nearline tables should all be world readable by the snoplus user, so unless you are updating the tables, you should be able to connect with the standard engine.

Reply:
Oops thanks I got mixed up with the snotdaq user who doesn't seem to be able to read from those tables